packages/styles/scss/ibm-dotcom-styles.scss

```diff
@@ -28,6 +28,7 @@
 @import 'patterns/listsection/listsection';
 @import 'patterns/pictogramarray/pictogramarray';
 @import 'patterns/simplelongform/simplelongform';
+@import 'patterns/simplelongform/simpleoverview';
```

Comment:
@RaphaelAmadeu is this the correct path?

Reply:
I've fixed the path on my latest commit, thanks!